package.json

```diff
@@ -3,7 +3,7 @@
   "version": "0.1.0",
   "private": true,
   "scripts": {
-    "dev": "next dev",
+    "dev": "next dev -H 0.0.0.0",
```

Comment:
not something I would recommend by default, because it makes the page network accessible, maybe we can create a different command for it?

Reply:
Ups, didn't want to commit this 😄